README.md

```diff
@@ -220,7 +220,28 @@ By default, grid size will be based on the content size. If you want it to stret
 
 ![Stretching grid demonstration.](docs/images/7-stretching.png)
 
-Keep in mind that when rendering components using flex inside ScrollView, you should set `{ flex: 1 }` as `contentContainerStyle` prop in order for it to stretch over the entire space. For complete examples take a look at source for the above [normal](examples/7-stretch-disabled.js) and [stretching](examples/7-stretch-enabled.js) examples.
```

Comment:
I think we should keep this warning somewhere, but note that the more appropriate way is to use scrollable. I am almost certain there will be users wondering why it isn't working, and if this single sentence saves them 10 minutes I think it is worth it. 

Possibly the best location would be at that part that we advocate using `ScrollView` manually for advanced use cases.

Reply:
You're right, I'll add this note. 👍

Reply:
Added.